wallet_support.json

```
@@ -496,6 +496,22 @@
       "uri": "https://twitter.com/john_zaprite"
       }
   },
+  {
+    "wallet": {
+      "name": "Wasabi Wallet",
+      "uri": "https://wasabiwallet.io/"
+    },
+    "scans_bip21": "yes",
+    "recognizes_lightning": "no",
```

```suggestion
    "recognizes_lightning": "n/a",
```
  
  I think this should be "N/A" because Wasabi isn't a lightning wallet. For a wallet that is on-chain only, disregarding the `lightning` param should be expected behavior.

---

Fixed that. 23b6842cb9fa5709c80bf4c15222a06dc3618c85